index.html

```diff
@@ -94,24 +110,24 @@ <h2 class="heading-underline">Upcoming events</h2>
 					<div class="articles">
 						<article class="article row">
```

Comment:
If you use a `row` class, the elements inside it *must* have `col-...` classes. The row is there to work with columns, if you use it by itself it will misalign the layout.